modules/nodes-pods-autoscaling-custom-gather.adoc

```diff
@@ -0,0 +1,185 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-gather_{context}"]
+= Gathering data about your cluster
+
+ifndef::openshift-origin[]
+When opening a support case, it is helpful to provide debugging information about your cluster to Red Hat Support.
+
+It is recommended that you provide the following information:
+
+* Data gathered using the `must-gather` tool.
+* The unique cluster ID.
+endif::openshift-origin[]
+
+You can use the `must-gather` tool to collect data about the Custom Metrics Autoscaler Operator and its components, including:
+
+* The `openshift-keda` namespace and its children objects.
+* The Custom Metric Autoscaler Operator installation objects.
+* The Custom Metric Autoscaler Operator CRD objects.
+
+The following command runs the `must-gather` tool for the Custom Metrics Autoscaler Operator:
+
+[source,terminal]
+----
+oc adm must-gather --image="$(oc get packagemanifests openshift-custom-metrics-autoscaler-operator \
+-n openshift-marketplace \
+-o jsonpath='{.status.channels[?(@.name=="stable")].currentCSVDesc. annotations.containerImage}')"
+----
+
+[NOTE]
+====
+The standard {product-title} must-gather command, `oc adm must-gather`, does not collect Custom Metrics Autoscaler Operator data.
+====
+
+.Example must-gather output for the Custom Metric Autoscaler:
```

Comment:
Good call on collapsible :) Does it make sense to move this to after the steps where the user runs the command?

Reply:
Done!